fireant/queries/sql_transformer.py

```diff
@@ -27,6 +25,7 @@
 
 @apply_special_cases
 def make_slicer_query_with_totals_and_references(
+    *,
```

Comment:
What does this mean?

Reply:
Adding a single asterisk forces you to pass all of the following arguments as keyword arguments. The reason I did this was to make code calling this function more clear, given the large number of positional arguments.